config/services/zabbix-agent.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Zabbix Agent</short>
+  <description>Allow services for Zabbix agent</description>
```

Comment:
The description should describe what Zabbix is.
e.g. "Zabbix is enterprise open source monitoring software for networks and applications"